pkg/download/automation/automation.go

```diff
@@ -78,6 +80,13 @@ func (d *Downloader) Download(projectName string, automationTypes ...config.Auto
 		for _, obj := range response.Results {
 
 			configId := obj.Id
+
+			if escaped, err := escapeJinjaTemplates(obj.Data); err != nil {
+				log.Warn("Failed to escape automation jinja for config %v (%s) - template needs manual adaptation: %v", configId, at.Resource, err)
```

Comment:
I'm not sure customers will be aware of what Jinja is, even though the Workflows docs mention it [0]

I'd suggest we use naming that fits the more general terms used in the docs

```suggestion
				log.Warn("Failed to escape automation templating expressions for config %v (%s) - template needs manual adaptation: %v", configId, at.Resource, err)
```


[0] https://www.dynatrace.com/support/help/shortlink/automations-reference

Reply:
changed